### ISSUE TITLE
Add a possibility to resize name column width in "Saved windows" window

### ISSUE THREAD

**Issue by ekasprzak** [khimaros/smart-auto-move#37]
Preferences content doesn't resize to the the window and in my case looks like this:
![image](https://user-images.githubusercontent.com/25803021/207307769-4020ec59-bb6e-4e47-b61b-14c1e87f97a9.png)

With many apps I'm not able to recognize the app, eg. "Org.gno..." - there are 4 instances, 4 apps and no idea which is which.
Another example - I use lot of PWA apps based on chrome.
Now, all of those are "Google-c...".

Maybe it's related to #19 ?

**Comment by khimaros** [comment]
yes, i'd consider this to fall under the work in #19 but i'm fine leaving this open until the parent issue is resolved.

**Comment by ChrisLauinger77** [comment]
Take a look at #56
